vue-components/src/components/HelloWorld.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-	<div class="wikit-HelloWorld">
+	<div class="wikit wikit-HelloWorld">
```

Comment:
```suggestion
	<div class="wikit HelloWorld">
```

What do you guys think about dropping the prefix?
I feel with the wikit class present at some point in the tree (and in the component library that tree should not be too deep), the component name should be descriptive enough to give people inspecting the DOM an idea where to start looking for the implementation (which in my eyes was the reason for adding the prefix).

OTOH with a strong prefix we are also shielding us from accidentally inheriting from someone trying to style all `.Button`s...

Reply:
> OTOH with a strong prefix we are also shielding us from accidentally inheriting from someone trying to style all .Buttons...

Yes, I think that's a valid concern.
I'm in favor of keeping it, even though it looks a bit strange to have `wikit wikit-..`